virtcontainers/qemu.go

```diff
@@ -673,6 +670,9 @@ func (q *qemu) setupVirtiofsd() (err error) {
 	}
 
 	err = cmd.Start()
+	if err != nil {
+		return fmt.Errorf("virtiofs daemon %v returned with error: %v", q.config.VirtioFSDaemon, err);
+	}
 	if err == nil {
 		q.state.VirtiofsdPid = cmd.Process.Pid
 	}
```

Comment:
Would you mind to make this code block more readable?
```
if err != nil {
}
if err == nil {
}
```

I'd suggest:
```
if err != nil {
    return ....
}

q.state.VirtiofsdPid = cmd.Process.Pid
```
```

Reply:
Done. Thanks for the reminder.